src/Akeneo/Pim/Automation/DataQualityInsights/back/Infrastructure/Connector/Tasklet/MarkEvaluation/MarkProductModelsEvaluationImpactedByUpdatedProductModelsTasklet.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Akeneo\Pim\Automation\DataQualityInsights\Infrastructure\Connector\Tasklet\MarkEvaluation;
+
+use Akeneo\Pim\Automation\DataQualityInsights\Application\ProductEvaluation\CreateCriteriaEvaluations;
+use Akeneo\Pim\Automation\DataQualityInsights\Domain\Query\ProductEvaluation\GetUpdatedEntityIdsQueryInterface;
+use Psr\Log\LoggerInterface;
+
+/**
+ * @copyright 2023 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+final class MarkProductModelsEvaluationImpactedByUpdatedProductModelsTasklet extends AbstractMarkEvaluationTasklet
```

Comment:
```suggestion
final class MarkProductModelsEvaluationImpactedByUpdateTasklet extends AbstractMarkEvaluationTasklet
```

or MarkProductModelsEvaluationImpactedByProductModelUpdateTasklet ?

The name you choose is also ok for me. You can keep like that if you want

Reply:
The name `MarkProductModelsEvaluationImpactedByUpdatedProductModelsTasklet` is too long and redundant at first sight, but it avoids any confusion with other use-cases in EE as `MarkProductModelsEvaluationImpactedByUpdatedAttributeLabelTasklet`
The problem with `MarkProductModelsEvaluationImpactedByUpdateTasklet` is that it doesn't tell what as been updated
When listed in the job definition, it's pretty clear to know what step do what: https://github.com/akeneo/pim-enterprise-dev/blob/31bd3de308ab21945141e7b119d79d893f863000/src/Akeneo/Pim/Automation/DataQualityInsights/back/Infrastructure/Symfony/Resources/config/jobs.yml#L20